### PR TITLE
Adding profiler test for WMT model, adding dependencies for WMT

### PR DESCRIPTION
**Pull request by RoshaniN**
1. Adding a profiler test - "flax-latest-wmt-profile-v3-8-1vm" which runs the WMT model and verifies the existence of a trace.
2. Including WMT model dependencies, i.e. correct version of protobuf ( to avoid incompatibility issues with TF ) and tensorflow_datasets.

( Issues are not related to this XL-ML test directly, but dependencies of WMT model itself.

Protobuf 3.20.* was needed to avoid tensorflow import error-
"TypeError: Descriptors cannot not be created directly.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower)."
 
 tensorflow_datasets import was needed because tensorflow_datasets is also imported in WMT. )

### CODE REVIEW

**Comment by RoshaniN** [comment]
https://buganizer.corp.google.com/issues/263413201 - logged a bug for the dependency issues.

**Comment by RoshaniN** [comment]
@will-cromar Could you take a look ?

**Review by will-cromar — APPROVED** [review]

File: tests/jax/latest/wmt.libsonnet [hunk 1]
```diff
@@ -33,10 +38,17 @@ local tpus = import 'templates/tpus.libsonnet';
...
+  local wmt_profiling = wmt {
+    local config = self,
+    testScript+:: |||
+      gsutil -q stat $(MODEL_DIR)/plugins/profile/*/*.xplane.pb
+    ||| % (self.scriptConfig {}),
   },
   configs: [
     wmt + functional + v2_8,
     wmt + convergence + v3_8 + timeouts.Hours(20),
+    wmt_profiling + profile + v3_8 + timeouts.Hours(1),
```
Comment (bvandermoon) [inline tests/jax/latest/wmt.libsonnet:52]: How long has the profiler test been taking to run? Just wondering if the timeout could be cut to something like 20-30 minutes instead

File: tests/jax/latest/wmt.libsonnet [hunk 2]
```diff
@@ -33,10 +38,17 @@ local tpus = import 'templates/tpus.libsonnet';
   },
   local wmt = common.runFlaxLatest {
     modelName:: 'wmt',
-    extraDeps+:: ['tensorflow_text sentencepiece'],
+    extraDeps+:: ['tensorflow_text sentencepiece', 'protobuf==3.20.*', 'tensorflow_datasets'],
```
Comment (bvandermoon) [inline tests/jax/latest/wmt.libsonnet:41]: Minor: seems like two formatting styles are being mixed here. tensorflow_text and sentencepiece are separated by space and the others are separated by commas

**Review by bvandermoon — APPROVED** [review]

File: tests/jax/latest/wmt.libsonnet (hunk 1 above)
Comment (RoshaniN) [reply tests/jax/latest/wmt.libsonnet:52]: Takes about 15-20 mins as per my observation. I didn't want to be too strict with the timeout here, as sometimes it takes ~10 mins to just provision the resources. 
There is an option to reduce it to 20-30 minutes - https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/timeouts.libsonnet#L23 . We could explore those in the next set of changes too.

File: tests/jax/latest/wmt.libsonnet (hunk 2 above)
Comment (RoshaniN) [reply tests/jax/latest/wmt.libsonnet:41]: I agree, I plan to remove the dependencies that has been added here soon after the bug is fixed.

**Review by bvandermoon — APPROVED** [review]

**Pull request merged**